include/CL/cl_exp_defined_builtin_kernels.h

```diff
@@ -209,4 +210,23 @@ typedef struct
   cl_int quality;
 } cl_dbk_attributes_exp_jpeg_encode;
 
+/* Name: "exp_onnx_inference"
```

Comment:
```suggestion
/* Name: "onnx_inference"
```
No need to have the 'exp' / khr / etx prefix if we go with the "all DBKs in the same extension" approach.

Reply:
That could be done in the future, but rn the pattern has been to set the name to the DBK string name and it could cause some confusion to deviate only on the onnx one.